app/services/hyrax/workflow/abstract_notification.rb

```diff
@@ -29,6 +30,16 @@ def message
           "#{title} (#{work_id}) was advanced in the workflow by #{user.user_key} and is awaiting approval #{comment}"
         end
 
+        # @return [document] the document (work) the the Abstract WorkFlow is creating a notification for
```

Comment:
The thing in the square brackets should be a class or list of classes.  So is it a `ActiveFedora::Base`?